htop.c

```diff
@@ -76,9 +59,9 @@ static void printHelpFlag(void) {
          "-V --version                    Print version info\n"
          "\n"
          "Long options may be passed with a single dash.\n\n"
-         "Press F1 inside " PACKAGE " for online help.\n"
-         "See 'man " PACKAGE "' for more information.\n",
-         stdout);
+         "Press F1 inside %s for online help.\n"
+         "See 'man %s' for more information.\n",
+         name, name, name);
```

Comment:
Why not use positional arguments here? Allows for re-use of the same argument.

Reply:
Ah, good idea!  Trying it out though, I get...

```
htop.c: In function ‘printHelpFlag’:
htop.c:64:10: warning: ISO C does not support %n$ operand number formats [-Wformat=]
   64 |          name);
      |          ^~~~
```